Web Inspector: Update COMPATIBILITY (macOS X.0, iOS X.0) comments with correct versions

rcaliman-apple opened the pull request:
#### f98fe54ddfaf74e0abf3618703ca676913ae1c33
<pre>
Web Inspector: Update COMPATIBILITY (macOS X.0, iOS X.0) comments with correct versions
<a href="https://bugs.webkit.org/show_bug.cgi?id=268176">https://bugs.webkit.org/show_bug.cgi?id=268176</a>
<a href="https://rdar.apple.com/121675559">rdar://121675559</a>

Reviewed by Patrick Angle.

Retroactively update backwards compatibility comments that were
missing corresponding version numbers for when they started applying.

Related to changes introduced in
- <a href="https://commits.webkit.org/267488@main">https://commits.webkit.org/267488@main</a>
- <a href="https://commits.webkit.org/264172@main">https://commits.webkit.org/264172@main</a> (relying on protocol changes in <a href="https://commits.webkit.org/261512@main">https://commits.webkit.org/261512@main</a> for the same version check)

* Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js:
(WI.CSSManager.prototype._mainResourceDidChange):
* Source/WebInspectorUI/UserInterface/Models/Canvas.js:
(WI.Canvas.fromPayload):
(WI.Canvas.prototype.sizeChanged):

Canonical link: <a href="https://commits.webkit.org/273700@main">https://commits.webkit.org/273700@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1d268769a528a1f298db7e68dfa297f9365cb29c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/36363 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/15317 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/38582 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/39079 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/32677 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/17761 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/12347 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/31337 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/36923 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/12928 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/32237 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/11338 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/11358 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/32455 "layout-tests (failure)") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/40323 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/32993 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/32816 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/37291 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/11570 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/9449 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/35387 "Passed tests") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/13280 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8250 "Built successfully and passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/12022 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/12430 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

patrickangle reviewed (approved):
For the future we should do these with the PR that adds the protocol. We should reference that commit in the commit message so relate this if we are doing it separately.

rcaliman-apple committed:
Web Inspector: Update COMPATIBILITY (macOS X.0, iOS X.0) comments with correct versions
https://bugs.webkit.org/show_bug.cgi?id=268176
rdar://121675559

Reviewed by Patrick Angle.

Retroactively update backwards compatibility comments that were
missing corresponding version numbers for when they started applying.

Related to changes introduced in
- https://commits.webkit.org/267488@main
- https://commits.webkit.org/264172@main (relying on protocol changes in https://commits.webkit.org/261512@main for the same version check)

* Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js:
(WI.CSSManager.prototype._mainResourceDidChange):
* Source/WebInspectorUI/UserInterface/Models/Canvas.js:
(WI.Canvas.fromPayload):
(WI.Canvas.prototype.sizeChanged):

Canonical link: https://commits.webkit.org/273700@main
